2-mandatory/1-weather-report.js

```diff
@@ -11,8 +11,14 @@
         - Hint: you can call the temperatureService function from your function
 */
 
+let temp;
 function getTemperatureReport(cities) {
-    // TODO
+    strings = [];
```

Comment:
string isn't a good choice of name here

Reply:
Can you suggest a better name? Or explain why `strings` isn't a good variable name?